No allocator thread panicked

I just built yofi from master as well as the 0.1.5 tag and when executing `yofi apps` I get the following error:

```
❯ yofi apps
thread '<unnamed>' panicked at 'Must provide allocators in no-stdlib code', /home/griff/.cargo/registry/src/github.com-1ecc6299db9ec823/brotli-decompressor-2.3.2/src/ffi/alloc_util.rs:185:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at 'Must provide allocators in no-stdlib code', /home/griff/.cargo/registry/src/github.com-1ecc6299db9ec823/brotli-decompressor-2.3.2/src/ffi/alloc_util.rs:185:13
thread '<unnamed>' panicked at 'Must provide allocators in no-stdlib code', /home/griff/.cargo/registry/src/github.com-1ecc6299db9ec823/brotli-decompressor-2.3.2/src/ffi/alloc_util.rs:185:13
thread '<unnamed>' panicked at 'Must provide allocators in no-stdlib code', /home/griff/.cargo/registry/src/github.com-1ecc6299db9ec823/brotli-decompressor-2.3.2/src/ffi/alloc_util.rs:185:13
thread '<unnamed>' panicked at 'Must provide allocators in no-stdlib code', /home/griff/.cargo/registry/src/github.com-1ecc6299db9ec823/brotli-decompressor-2.3.2/src/ffi/alloc_util.rs:185:13
[2022-04-27][20:48:47][yofi][ERROR] panic: Must provide allocators in no-stdlib code
```
I tracked down the crate generating the error. It is a dependency of `allsorts_no_std` which is a dependency of `rust-fontconfig`. The `allsorts_no_std` crate has `brotli-decompressor` as a dependency with default features off making it no_std and requiring that an allocator be set. It looks like `allsorts_no_std` may have moved to `allsorts` and likely might just need to be updated.

Let me know if you have any questions, need additional info, or would like me to assist in troubleshooting further.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,7 +13,10 @@ ttf-parser = { version = "0.15.0", optional = true }
 
 fontdue = { version = "0.7.2", optional = true }
 rust-fontconfig = { version = "0.1.5", optional = true }
+# Not used directly by yofi. Added to ensure brotli-decompressor is built with std support for rust-fontconfig
+brotli-decompressor = { version = "*", optional = true }
 encoding_rs = "=0.8.28"
+
 
 anyhow = "1.0.51"
 wayland-protocols = { version = "0.29.1", default-features = false, features = ["unstable_protocols", "client"] }
@@ -53,7 +56,7 @@ lto = true
 
 [features]
 default = ["font-fontdue"]
-font-fontdue = ["fontdue", "rust-fontconfig"]
+font-fontdue = ["fontdue", "rust-fontconfig", "brotli-decompressor"]
 font-fontkit = ["font-kit", "pathfinder_geometry", "raqote/text", "ttf-parser"]
 
 [dev-dependencies]

